test/types/enum/lydia/declaredInClass-initializer.future

```diff
@@ -1,8 +1,8 @@
-bug: enums declared in class scope can't have fields use their type
+bug: enums declared in class scope don't work with initializers yet
```

Comment:
Is there an Issue associated with this future?

Reply:
Yeah, #6871